cloudbuild.yaml

```diff
@@ -275,21 +277,19 @@ artifacts:
             - examples/backendclient/backendclient
             - test/cmd/clientloadgen/clientloadgen
             - test/cmd/frontendclient/frontendclient
-# TODO: Enable when the Dockerfiles are hermetic.
 images:
-#- 'gcr.io/$PROJECT_ID/openmatch-backendapi:${_OM_VERSION}-${SHORT_SHA}'
-#- 'gcr.io/$PROJECT_ID/openmatch-frontendapi:${_OM_VERSION}-${SHORT_SHA}'
-#- 'gcr.io/$PROJECT_ID/openmatch-mmforc:${_OM_VERSION}-${SHORT_SHA}'
-#- 'gcr.io/$PROJECT_ID/openmatch-mmlogicapi:${_OM_VERSION}-${SHORT_SHA}'
-# Does not build
-#- 'gcr.io/$PROJECT_ID/openmatch-evaluator-simple:${_OM_VERSION}-${SHORT_SHA}'
-#- 'gcr.io/$PROJECT_ID/openmatch-cs-func-simple:${_OM_VERSION}-${SHORT_SHA}'
-#- 'gcr.io/$PROJECT_ID/openmatch-go-func-simple:${_OM_VERSION}-${SHORT_SHA}'
-#- 'gcr.io/$PROJECT_ID/openmatch-backendclient:${_OM_VERSION}-${SHORT_SHA}'
-#- 'gcr.io/$PROJECT_ID/openmatch-clientloadgen:${_OM_VERSION}-${SHORT_SHA}'
-#- 'gcr.io/$PROJECT_ID/openmatch-frontendclient:${_OM_VERSION}-${SHORT_SHA}'
-#substitutions:
-#    _OM_VERSION: 0.4.0
+- 'gcr.io/$PROJECT_ID/openmatch-backendapi:${_OM_VERSION}-${SHORT_SHA}'
+- 'gcr.io/$PROJECT_ID/openmatch-frontendapi:${_OM_VERSION}-${SHORT_SHA}'
+- 'gcr.io/$PROJECT_ID/openmatch-mmforc:${_OM_VERSION}-${SHORT_SHA}'
+- 'gcr.io/$PROJECT_ID/openmatch-mmlogicapi:${_OM_VERSION}-${SHORT_SHA}'
+- 'gcr.io/$PROJECT_ID/openmatch-evaluator-simple:${_OM_VERSION}-${SHORT_SHA}'
+- 'gcr.io/$PROJECT_ID/openmatch-cs-func-simple:${_OM_VERSION}-${SHORT_SHA}'
```

Comment:
I do not see any Dockerfile change to the C# sample Dockerfile in this CL. I am not familiar with the build mechanism for it - but seems different from the rest of these. Do we know if it builds and is still a valid example?

Reply:
The build happens as part of Docker Image: openmatch-cs-func-simple.
This section is to upload to the build artifacts (docker images and binaries) to gcr.io and GCS.